src/pages/workspaces/workspace/WorkspaceContainer.test.ts

```diff
@@ -0,0 +1,109 @@
+import { render, screen, within } from '@testing-library/react';
+import { h } from 'react-hyperscript-helpers';
+import { WorkspaceContainer } from 'src/pages/workspaces/workspace/WorkspaceContainer';
+import { defaultAzureWorkspace, defaultGoogleWorkspace } from 'src/testing/workspace-fixtures';
+
+// Mocking for Nav.getLink
+jest.mock('src/libs/nav', () => ({
+  ...jest.requireActual('src/libs/nav'),
+  getLink: jest.fn(() => '/'),
+}));
```

Comment:
Ideally, mocks would have type safety.

```suggestion
type NavExports = typeof import('src/libs/nav');
jest.mock('src/libs/nav', (): NavExports => ({
  ...jest.requireActual<NavExports>('src/libs/nav'),
  getLink: jest.fn(() => '/'),
}));
```